discord_bot/chat_handler.py

```
@@ -542,18 +542,33 @@ async def on_message(message):
             )
             logger.info(f"[chatbot] Context prompt built, length: {len(prompt)} characters")
 
+            # Process attachments - images go to model, others get noted
             images = []
+            non_image_attachments = []
+            
             if message.attachments:
                 for attachment in message.attachments:
                     if attachment.content_type and attachment.content_type.startswith("image/"):
                         images.append(Image(url=attachment.url))
                         logger.info(f"[chatbot] Found image attachment: {attachment.url}")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)